### PR TITLE
Deprecated content quick wins

### PR DESCRIPTION
**Pull request by OmriItzhak**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes:  [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-6528)

## Description
Deprecate old playbooks and replace deprecated commands with new commands.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ArcherRSA/Playbooks/playbook-Archer_initiate_incident.yml [hunk 1]
```diff
@@ -2,8 +2,10 @@ id: Archer initiate incident
 version: -1
 name: Archer initiate incident
 fromversion: "5.0.0"
+deprecated: true
 starttaskid: "0"
-description: "initiate Archer incident"
+description: "Deprecated. Use the `archer-get-file` command directly instead.
```
Comment (altmannyarden) [inline Packs/ArcherRSA/Playbooks/playbook-Archer_initiate_incident.yml:7]: How get-file replacing "initiate Archer incident"?

Comment (altmannyarden) [reply Packs/ArcherRSA/Playbooks/playbook-Archer_initiate_incident.yml:7]: oh, the only task in the playbook is to get a file?

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Execute_snippet_code_script.yml [hunk 2]
```diff
@@ -1,58 +1,57 @@
-description: Initiates a new endpoint script execution action using the provided snippet
-  code and retrieves the file results.
 id: Cortex XDR - Execute snippet code script
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Execute_snippet_code_script.yml:1]: As far as I know - the developers change those commands with polling inside the command. so the playbook with polling checks isnt needed. (we can deprecate the playbook and use the command instead). 

Can you please check?

File: Packs/ArcherRSA/Playbooks/playbook-Archer_initiate_incident.yml (hunk 1 above)
Comment (OmriItzhak) [reply Packs/ArcherRSA/Playbooks/playbook-Archer_initiate_incident.yml:7]: Yes. The Playbook contains only one task with the deprecated command `archer-get-file`.

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_check_file_existence.yml [hunk 3]
```diff
@@ -46,7 +47,7 @@ tasks:
       version: -1
       name: Run script file exists
       description: Initiates a new endpoint script execution to check if file exists.
-      script: '|||xdr-file-exist-script-execute'
+      script: '|||xdr-run-script-file-exists'
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_check_file_existence.yml:50]: here as well. (I guess for all XDR playbooks here)

File: Packs/McAfee-TIE/Playbooks/playbook-Search_Endpoints_By_Hash_-_TIE.yml [hunk 4]
```diff
@@ -102,86 +102,6 @@ tasks:
...
     isoversize: false
     isautoswitchedtoquietmode: false
-  "2":
-    id: "2"
-    taskid: b098fed0-d1ae-418c-8a37-756e1ae0cb37
-    type: condition
-    task:
-      id: b098fed0-d1ae-418c-8a37-756e1ae0cb37
-      version: -1
-      name: Is ePO enabled?
```
Comment (altmannyarden) [inline Packs/McAfee-TIE/Playbooks/playbook-Search_Endpoints_By_Hash_-_TIE.yml:112]: Why you removed those tasks?

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Execute_snippet_code_script.yml [hunk 5]
```diff
@@ -44,8 +45,9 @@ tasks:
       id: 516d4dea-8b72-4ecf-80e3-762ab62e503d
       version: -1
       name: Run snippet code script
-      description: Initiates a new endpoint script execution action using the provided snippet code.
-      script: '|||xdr-snippet-code-script-execute'
+      description: Initiates a new endpoint script execution action using the provided
+        snippet code.
+      script: '|||xdr-run-snippet-code-script'
```
[the remark this answers is not in the document]
Comment (OmriItzhak) [reply Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Execute_snippet_code_script.yml:50]: xdr-run-snippet-code-script is the original command. I have already reverted this line.

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_check_file_existence.yml (hunk 3 above)
Comment (OmriItzhak) [reply Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_check_file_existence.yml:50]: xdr-run-script-file-exists is the original command. I have already reverted this line.

File: Packs/McAfee-TIE/Playbooks/playbook-Search_Endpoints_By_Hash_-_TIE.yml (hunk 4 above)
Comment (OmriItzhak) [reply Packs/McAfee-TIE/Playbooks/playbook-Search_Endpoints_By_Hash_-_TIE.yml:112]: `epo v1` pack has reached its end of life and is no longer supported. End-of-Life Date: Aug 01, 2022
I removed from the playbook the `epo v1` commands.
The condition task for checking if the `McAfee epo v2` integration is enabled and 'epo-find-system (McAfee ePO v2)' command already exists in the playbook.

File: Packs/McAfee-TIE/ReleaseNotes/2_0_7.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+#### Playbooks
+##### TIE - IOC Hunt
+- Updated deprecated command from *tie-file-references (McAfee Threat Intelligence Exchange)* to *tie-file-references*.
```
Comment (ShirleyDenkberg) [inline Packs/McAfee-TIE/ReleaseNotes/2_0_7.md:3]:
```suggestion
- Updated the deprecated command from ***tie-file-references (McAfee Threat Intelligence Exchange)*** to ***tie-file-references***.
```

File: Packs/CommonPlaybooks/ReleaseNotes/2_3_70.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+#### Playbooks
+
+##### Block Account - Generic v2
+Updated deprecated command from ***panorama-register-user-tag*** to ***pan-os-register-user-tag***.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_3_70.md:4]:
```suggestion
Updated the deprecated command from ***panorama-register-user-tag*** to ***pan-os-register-user-tag***.
```

File: Packs/McAfee-TIE/ReleaseNotes/2_0_7.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+- Removed command *EPOFindSystem*. This command is part of the deprecated *McAfee ePO* pack, which has reached its end of life
```
Comment (ShirleyDenkberg) [inline Packs/McAfee-TIE/ReleaseNotes/2_0_7.md:4]:
```suggestion
- Removed the ***EPOFindSystem*** command. This command is part of the deprecated **McAfee ePO** pack, which has reached its end of life
```

File: Packs/McAfee-TIE/ReleaseNotes/2_0_7.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
...
+(EOL) and is no longer supported. Please use the integration *McAfee ePO v2* instead.
```
Comment (ShirleyDenkberg) [inline Packs/McAfee-TIE/ReleaseNotes/2_0_7.md:5]:
```suggestion
(EOL) and is no longer supported. Please use the integration *McAfee ePO v2* instead.
```
```suggestion
(EOL) and is no longer supported. Use the **McAfee ePO v2** integration  instead.
```

File: Packs/McAfee-TIE/ReleaseNotes/2_0_7.md [hunk 10]
```diff
@@ -0,0 +1,10 @@
...
+
+##### Search Endpoints By Hash - TIE
+- Updated deprecated command from *tie-file-references (McAfee Threat Intelligence Exchange)* to *tie-file-references*.
```
Comment (ShirleyDenkberg) [inline Packs/McAfee-TIE/ReleaseNotes/2_0_7.md:8]:
```suggestion
- Updated the deprecated command from ***tie-file-references (McAfee Threat Intelligence Exchange)*** to ***tie-file-references***.
```

File: Packs/McAfee-TIE/ReleaseNotes/2_0_7.md [hunk 11]
```diff
@@ -0,0 +1,10 @@
...
+- Removed command *EPOFindSystem*. This command is part of the deprecated *McAfee ePO* pack, which has reached its end of life
```
Comment (ShirleyDenkberg) [inline Packs/McAfee-TIE/ReleaseNotes/2_0_7.md:9]:
```suggestion
- Removed the ***EPOFindSystem*** command. This command is part of the deprecated **McAfee ePO** pack, which has reached its end of life
```

File: Packs/McAfee-TIE/ReleaseNotes/2_0_7.md [hunk 12]
```diff
@@ -0,0 +1,10 @@
...
+(EOL) and is no longer supported. Please use the integration *McAfee ePO v2* instead.
```
Comment (ShirleyDenkberg) [inline Packs/McAfee-TIE/ReleaseNotes/2_0_7.md:10]:
```suggestion
(EOL) and is no longer supported. Use the **McAfee ePO v2** integration instead.
```

File: Packs/PrismaSaasSecurity/ReleaseNotes/2_0_21.md [hunk 13]
```diff
@@ -0,0 +1,4 @@
+#### Playbooks
+
+##### Saas Security - Incident Processor
+Updated deprecated command from ***send-mail (EWS Mail Sender)*** to ***send-mail***.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaSaasSecurity/ReleaseNotes/2_0_21.md:4]:
```suggestion
Updated the deprecated command from ***send-mail (EWS Mail Sender)*** to ***send-mail***.
```

File: Packs/Tanium/ReleaseNotes/1_0_25.md [hunk 14]
```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Tanium Demo Playbook
+
+- Updated deprecated command from ***tn-get-saved-question (Tanium)*** to ***tn-get-saved-question-metadata***.
```
Comment (ShirleyDenkberg) [inline Packs/Tanium/ReleaseNotes/1_0_25.md:6]:
```suggestion
- Updated the deprecated command from ***tn-get-saved-question (Tanium)*** to ***tn-get-saved-question-metadata***.
```

File: Packs/Tanium/ReleaseNotes/1_0_25.md [hunk 15]
```diff
@@ -0,0 +1,7 @@
...
+- Updated deprecated command from ***tn-ask-question (Tanium)*** to ***tn-ask-question***.
```
Comment (ShirleyDenkberg) [inline Packs/Tanium/ReleaseNotes/1_0_25.md:7]:
```suggestion
- Updated the deprecated command from ***tn-ask-question (Tanium)*** to ***tn-ask-question***.
```

File: Packs/qualys/Playbooks/playbook-Vulnerability_Management__-_Qualys_(Job)_README.md [hunk 16]
```diff
@@ -1,48 +1,56 @@
-Manages vulnerabilities using the latest Qualys report.
+Deprecated. Use the `Vulnerability Management - Qualys (Job) - V2` playbook instead.
```
Comment (ShirleyDenkberg) [inline Packs/qualys/Playbooks/playbook-Vulnerability_Management__-_Qualys_(Job)_README.md:1]:
```suggestion
Deprecated. Use the **Vulnerability Management - Qualys (Job) - V2** playbook instead.
```

File: Packs/qualys/Playbooks/playbook-Vulnerability_Management__-_Qualys_(Job)_README.md [hunk 17]
```diff
@@ -1,48 +1,56 @@
...
-You can define the minimum severity `minSeverity` that incidents are created for.
+You can define the minimum severity (minSeverity) that incidents are created for.
 Duplicate incidents are not created for the same asset ID and QID.
 
 This playbook is a part of a series of playbooks for Qualys vulnerability management and remediation.
 For this series of playbooks to run successfully, create a Job and do the following:
 1. Assign this playbook to the Job
 2. Enter the Qualys XML report name into the "Details" field
-3. Associate the "Vulnerability" type incident to the `Vulnerability Handling - Qualys` playbook.
+3. Associate the "Vulnerability" type incident to the "Vulnerability Handling - Qualys" playbook.
```
Comment (ShirleyDenkberg) [inline Packs/qualys/Playbooks/playbook-Vulnerability_Management__-_Qualys_(Job)_README.md:13]:
```suggestion
3. Associate the *Vulnerability* type incident to the **Vulnerability Handling - Qualys** playbook.
```

File: Packs/qualys/Playbooks/playbook-Vulnerability_Management__-_Qualys_(Job)_README.md [hunk 18]
```diff
@@ -1,48 +1,56 @@
...
 ---
 
-| **Name** | **Description** | **Default Value** | **Source** | **Required** |
-| --- | --- | --- | --- | --- |
-| QualysReportTitle | The report title as it appears in Qualys. Has to be in XML format.  | details | incident | Required |
-| MinSeverity | The minimum Qualys severity (1 -5) to create incidents for. | 3 | - | Required |
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| QualysReportTitle | The report title as it appears in Qualys.<br/>Has to be in XML format.  | incident.details | Required |
+| MinSeverity | The minimum Qualys severity \(1 -5\) to create incidents for | 3 | Required |
```
Comment (ShirleyDenkberg) [inline Packs/qualys/Playbooks/playbook-Vulnerability_Management__-_Qualys_(Job)_README.md:45]:
```suggestion
| MinSeverity | The minimum Qualys severity \(1 -5\) to create incidents for. | 3 | Required |
```

File: Packs/qualys/ReleaseNotes/1_2_9.md [hunk 19]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Vulnerability Management - Qualys (Job)
+
+- Deprecated. Use the `Vulnerability Management - Qualys (Job) - V2` playbook instead.
```
Comment (ShirleyDenkberg) [inline Packs/qualys/ReleaseNotes/1_2_9.md:6]:
```suggestion
Deprecated. Use the **Vulnerability Management - Qualys (Job) - V2** playbook instead.
```

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**